src/Command/InvoiceCreateCommand.php

```diff
@@ -197,6 +197,16 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             @trigger_error('The "set-exported" option of kimai:invoice:create command has no meaning anymore, it will be removed soon', E_USER_DEPRECATED);
         }
 
+        $overwriteTemplate = null;
+        if (null !== $input->getOption('template')) {
+            $overwriteTemplate = $this->getCommandLineArgumentTemplateForCustomer($input->getOption('template'));
```

Comment:
```suggestion
        if (null !== ($tplOption = $input->getOption('template')) && is_string($tplOption)) {
            $overwriteTemplate = $this->getCommandLineArgumentTemplateForCustomer($tplOption);
```
That should be enough to get rid of the added phpstan error

Reply:
Actually, we do allow to enter the templateId, so `is_string` would exclude that.

Reply:
Command line options are IMO always strings or null/false if not given. 
Even a template ID should be a numerical string.

Reply:
If I do this
```
        $overwriteTemplate = null;
        if (null !== ($tplOption = $input->getOption('template'))) {
            if (\is_string($tplOption)){
                $tplOption = \strval($tplOption);
                $overwriteTemplate = $this->getCommandLineArgumentTemplateForCustomer($tplOption);
                if (null === $overwriteTemplate){
                    $io->error('Invalid invoice template name or id given');

                    return Command::FAILURE;
                }
            } else {
                $io->error('Option --template must be string or int');

                return Command::FAILURE;
            }
        }
```
and enter in the test the template ID as int, then I get the error message of the `else` case. 

In symfony the return type of `getOption` is mixed, it can be type set by `setDefault` to `string|float|int|...`, however it can also be an array of strings.. So, I'm not sure what I'm facing here... 

That's why I added an `is_int` for the `if` clause additionally to the `is_string`. Any better idea is welcome...